tests/load_interrupt/run.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -eu
+
+cur=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+source $cur/../_utils/test_prepare
+WORK_DIR=$TEST_DIR/$TEST_NAME
+
+COUNT=200
+function prepare_datafile() {
+    for i in $(seq 2); do
+        data_file="$WORK_DIR/db$i.prepare.sql"
+        echo 'DROP DATABASE if exists load_interrupt;' >> $data_file
+        echo 'CREATE DATABASE load_interrupt;' >> $data_file
+        echo 'USE load_interrupt;' >> $data_file
+        echo "CREATE TABLE t$i(i TINYINT, j INT UNIQUE KEY);" >> $data_file
+        for j in $(seq $COUNT); do
+            echo "INSERT INTO t$i VALUES ($i,${j}000$i),($i,${j}001$i);" >> $data_file
+        done
+    done
+}
+
+function run() {
+    THRESHOLD=1024
+    prepare_datafile
+
+    run_sql_file $WORK_DIR/db1.prepare.sql $MYSQL_HOST1 $MYSQL_PORT1
+    run_sql_file $WORK_DIR/db2.prepare.sql $MYSQL_HOST2 $MYSQL_PORT2
+
+    export GO_FAILPOINTS="github.com/pingcap/dm/loader/LoadExceedOffsetExit=return($THRESHOLD)"
+
+    run_dm_worker $WORK_DIR/worker1 $WORKER1_PORT $cur/conf/dm-worker1.toml
+    run_dm_worker $WORK_DIR/worker2 $WORKER2_PORT $cur/conf/dm-worker2.toml
+    run_dm_master $WORK_DIR/master $MASTER_PORT $cur/conf/dm-master.toml
+
+    check_port_alive $MASTER_PORT
+    check_port_alive $WORKER1_PORT
+    check_port_alive $WORKER2_PORT
+
+    cd $cur && GO111MODULE=on go build -o bin/dmctl && cd -
+    $cur/bin/dmctl "$cur/conf/dm-task.yaml"
+
+    check_port_offline $WORKER1_PORT 20
+    check_port_offline $WORKER2_PORT 20
+
+    run_sql "SELECT count(*) from dm_meta.test_loader_checkpoint where cp_schema = '$TEST_NAME' and offset < $THRESHOLD" $TIDB_PORT
```

Comment:
can we estimate how many rows are there?

Reply:
added